docs/words.txt

```diff
@@ -18,8 +19,11 @@ localhost
 memcache
 Memcache
 memcached
+oauth
 Queing
 Redis
+Reddit
+reddit
```

Comment:
we're technically not supposed to have any lower case reddits anymore... though I kinda love 'em still.